src/main/telemetry/jetiexbus.c

```diff
@@ -359,9 +470,7 @@ void handleJetiExBusTelemetry(void)
         }
 
         if ((jetiExBusRequestFrame[EXBUS_HEADER_DATA_ID] == EXBUS_EX_REQUEST) && (jetiExBusCalcCRC16(jetiExBusRequestFrame, jetiExBusRequestFrame[EXBUS_HEADER_MSG_LEN]) == 0)) {
-            // switch to TX mode
             if (serialRxBytesWaiting(jetiExBusPort) == 0) {
-                serialSetMode(jetiExBusPort, MODE_TX);
```

Comment:
Why is this removed? I suspect this will break setups that use a separate TX and RX pins.

Reply:
No, this was only necessary because RX & TX are connected with a resistor.
When sending data, they are also received by the RX, so switching between sending and receiving is explicitly done.

This now happens in terms of hardware, as soon as the transmission is finished, it is switched back to reception.

Reply:
The issue I have with this is that many users with existing set ups will have their RX connected to RX and TX, with a resistor in between, and will not want to change their wiring. We need to make sure that these existing setups are still working after this change.

Reply:
I know what you mean. As I said, it works with unchanged wiring. But I'll record the signals and upload them in the PR #6767